### PR TITLE
Add test cases for errors

### PR DESCRIPTION
**Pull request by jhaals**

### CODE REVIEW

**Comment by jhaals** [comment]
@grobie Please take a look

File: pkg/yopass/yopass_test.go [hunk 1]
```diff
@@ -147,6 +148,21 @@ func TestEncrypt(t *testing.T) {
...
+
+func (invalidFile) Read(p []byte) (n int, err error) { return 0, fmt.Errorf("Broken I/O") }
+
+func TestEncryptWithInvalidFile(t *testing.T) {
+	_, err := yopass.Encrypt(invalidFile{}, "somekey")
+	if err == nil {
+		t.Fatal("expected error, got none")
+	}
+	want := "could not copy data: Broken I/O"
+	if err.Error() != "could not copy data: Broken I/O" {
```
Comment (grobie) [inline pkg/yopass/yopass_test.go:161]: I guess you could use the `want` variable here?

**Review by grobie — APPROVED** [review]
Thanks a lot!

File: pkg/yopass/yopass_test.go (hunk 1 above)
Comment (jhaals) [reply pkg/yopass/yopass_test.go]: Yes thanks! 👍

**Pull request merged**